data/model/repository.py

```diff
@@ -266,6 +271,7 @@ def get_visible_repositories(
         )
         .switch(Repository)
         .join(Namespace, on=(Repository.namespace_user == Namespace.id))
+        .where(Repository.state != RepositoryState.MARKED_FOR_DELETION)
```

Comment:
Should we add a method for retrieving deleted repositories?

Reply:
I don't think its necessary for the time being